sdk/identity/identity/package.json

```diff
@@ -85,8 +85,10 @@
     "@azure/logger": "^1.0.0",
     "@azure/msal-node": "1.0.0-beta.3",
     "@opentelemetry/api": "^0.10.2",
+    "@types/express": "^4.16.0",
     "axios": "^0.21.1",
     "events": "^3.0.0",
+    "express": "^4.16.3",
```

Comment:
Please check with @jonathandturner if this was left as dev dependency on purpose. As noted in #13103, this increases the bundle size significantly

cc @xirzec

Reply:
@ramya-rao-a I was curios about this as well, it looks weird to depend on `express` in the SDK. Either way, it is my understanding that if someone tries to use interactive browser credentials in identity now and they do not happen to have `express` installed, they will get an error because it is being used in the source code. So I believe this is a bug in a shipped package and have to be fixed by at least adding express as a dependency for the time being. That being said, I understand the bundle size concern and for now I would recommend to the customer to mark `express` as external in their webpack configs if they do not need the interactive browser creds module.

Reply:
If I recall correctly, I added it so that interactive browser would work correctly. Currently, IBC isn't optional functionality, it's part of the core features of identity.

We could look into writing that part of the IBC functionality differently.

Reply:
peerDependency could be an option here.

Reply:
Taking a runtime dep on express for IBC feels bad. I'd like to rewrite if possible. In a past life I wrote a very small HTTP server in C# in order to do code coverage, so it's not as bad/crazy as it sounds.

Reply:
@xirzec Would you say we not take this change and revisit this eslint plugin after the IBC rewrite? cc @sadasant, @maorleger

Reply:
@ramya-rao-a if we are not adding `express` as a dep, I think we may as well remove IBC all together until we have a good implementation for it.

Reply:
is [this](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/identity/identity/src/credentials/interactiveBrowserCredential.ts#L115) the only usage of express for IBC? Maybe we can replace it with vanilla JS counterparts instead as a quick(ish) fix? 

Like would it work if we replaced express with http+url (for query string)? 

I'm curious about longer term - Since this in the browser I think I'm missing how it works today. 😄

Reply:
I'm a little uncomfortable shipping a GA version of identity that depends on express, but since it's not breaking to remove a dependency, I would be fine shipping as is and refactoring this during the month so we can release a version that doesn't depend on express in March.

Reply:
Looks like we already shipped 1.2.3 and the next Identity release will be preview. So, lets go ahead with this and ensure we do the refactoring for IBC this month

Reply:
Sounds good. I will merge the PR as is. Perhaps we should consider a patch release with this PR change.